etchosts/etchosts.go

```diff
@@ -69,32 +68,12 @@ func Drop(path string) {
 
 // Build function
 // path is path to host file string required
-// IP, hostname, and domainname set main record leave empty for no master record
 // extraContent is an array of extra host records.
-func Build(path, IP, hostname, domainname string, extraContent []Record) error {
```

Comment:
I would not change this package public method. I do not think we should remove this functionality just because current libnetwork code does not pass use it. Because we may end up reusing it in future. In fact, in older version this method was being called in a different part of the code and the IP was being passed: https://github.com/docker/docker/blob/v1.9.1/vendor/src/github.com/docker/libnetwork/sandbox.go#L630

IIUC, the change in `func (sb *sandbox) updateHostsFile(ifaceIP string) error {` is all what is needed to address this issue.